staging/src/k8s.io/apiserver/pkg/server/filters/timeout_test.go

```diff
@@ -258,10 +258,11 @@ func TestErrConnKilled(t *testing.T) {
 		t.Fatal("expected to receive an error")
 	}
 
-	// we should only get one line for this, not the big stack from before
 	capturedOutput := readStdErr()
-	if strings.Count(capturedOutput, "\n") != 1 {
-		t.Errorf("unexpected output captured actual = %v", capturedOutput)
+
+	// We don't expect stack trace from the panic to be included in the log.
+	if isStackTraceLoggedByRuntime(capturedOutput) {
```

Comment:
have you identified a test that fails on the previous logic?
from what I have seen there no tests in this pkg that would use `FinishRequest` method.

Reply:
@p0lyn0mial it's coming from this test - https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/filters/timeout_test.go#L136